### ISSUE TITLE
Some include files still point to http://fargo.io

### ISSUE THREAD

**Issue by SparkyT** [scripting/river5#23]
The following 4 files in the repo and on jsdelivr.net appear to have document.write calls to add scripts from http://fargo.io, not the new //cdn.jsdelivr.net locations causing https instances of river5 to not work as expected

- includes/basicincludes2/beep.js
- includes/riverBrowserIncludes/emojify.js
- includes/riverBrowserIncludes/riverbrowser.js
- includes/misc/riverbrowser.js

**Comment by SparkyT** [comment]
Good point - the document.write puts references to http resources which on https instances of river5 results in mixed content warnings and them being blocked.  River5 then doesn't display rivers (see screenshot)
![river5-docwrite](https://user-images.githubusercontent.com/4244537/39232179-5b420012-4864-11e8-9c42-1bc8f24d54e7.PNG)

**Comment by scripting** [comment]
Now I understand.

I can't test this stuff myself because I don't have an HTTPS server to
deploy to.

I'll work on this today. I had planned to get to this eventually. The
problem is there are other apps I have that depend on these files, and
depend on the document.write statements.

River5 doesn't depend on them, so I figured the document.writes would
generate warnings, but they wouldn't stop the app from running. Now I see
it's otherwise.

It would be helpful if you could quickly test versions, then I can iterate.

Back in a bit with something for you to try.

**Comment by scripting** [comment]
OK, to do this final (I hope) pass, I've set up a virgin River5 on my desktop.

I'm looking at the home page in the browser. These are the files that are included at the top of the page.

//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/alertdialog.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/app.css
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/app.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/appprefs.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/askdialog.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/beep.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/bootstrap.css
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/bootstrap.min.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/confirmdialog.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/font-awesome.min.css"/>
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/ga.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/jBeep.min.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/jquery-1.9.1.min.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/menus.css
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/menus.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/strftime.js
//cdn.jsdelivr.net/npm/river5/includes/basicincludes2/utils.js
//cdn.jsdelivr.net/npm/river5/includes/riverBrowserIncludes/api.js
//cdn.jsdelivr.net/npm/river5/includes/riverBrowserIncludes/dateFormat.js
//cdn.jsdelivr.net/npm/river5/includes/riverBrowserIncludes/emojify.css
//cdn.jsdelivr.net/npm/river5/includes/riverBrowserIncludes/emojify.js
//cdn.jsdelivr.net/npm/river5/includes/riverBrowserIncludes/markdownConverter.js
//cdn.jsdelivr.net/npm/river5/includes/riverBrowserIncludes/outlinebrowser.js
//cdn.jsdelivr.net/npm/river5/includes/riverBrowserIncludes/riverbrowser.css
//cdn.jsdelivr.net/npm/river5/includes/riverBrowserIncludes/riverbrowser.js
//fonts.googleapis.com/css?family=Rancho
//fonts.googleapis.com/css?family=Ubuntu

So these all look cool. The question is what document.writes do each of them trigger?

**Comment by scripting** [comment]
BTW, I have steps-to-reproduce, and through the magic of jsdelivr, I believe I will be able to fix this problem without you having to test. 

I noticed the URL in your screen shot, a familiar one, with https instead of http.

https://newriver5.glitch.me/

Boom, that page doesn't load and when I look at the console there are the errors.

**Comment by scripting** [comment]
Here are the files the browser complains about.

http://fargo.io/code/shared/jBeep.min.js
'http://fargo.io/code/emojify/emojify.js
'http://fargo.io/code/node/shared/utils.js
'http://api.nodestorage.io/api.js
'http://fargo.io/cms/dateFormat.js
'http://fargo.io/code/browsers/outlinebrowser.js
'http://counter2.fargo.io:5337/counter
http://fargo.io/code/ubuntuFont.css
http://fargo.io/code/fontAwesome/css/font-awesome.min.css
http://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic
http://fargo.io/code/browsers/riverbrowser.css

**Comment by scripting** [comment]
@SparkyT -- in order to get this debugged, I need you to do an npm update on the server, to get the latest version of davereader. I just changed the default URL of the homepage and the dashboard page My CDN is being fussy about getting the latest version, this way is fairly foolproof. 

Thanks for all your help with this. Very cool. ;-)

**Comment by SparkyT** [comment]
@scripting - no problem, it's interesting being involved in something that someone else has created.

`npm update` is done and it looks as though https://newriver5.glitch.me/ is now working.

Given we have the chance now before it is really publicised do you want to change the glitch name from newriver5 to something else?

**Comment by scripting** [comment]
@sparkyt -- this is great. ;-)

1. Re the glitch name, obviously River5 is the best choice.

2. Has someone already taken that name? If so any chance of getting them to relinquish it?

3. Could you change "World News" to just "World" -- the news part is implicit, and it'll look more balanced if it's shorter. 

4. I'm surprised it works. There are still errors which I will fix, in the console. In the meantime it works, which is good. ;-)
